------------------------------------------------------------
/usr/bin/pip3 run on Thu Feb 14 18:25:50 2019
Downloading/unpacking urlparse4
  Getting page https://pypi.python.org/simple/urlparse4/
  URLs to search for versions for urlparse4:
  * https://pypi.python.org/simple/urlparse4/
  Analyzing links from page https://pypi.org/simple/urlparse4/
    Found link https://files.pythonhosted.org/packages/36/86/c8fa7c9534d1560f38455154bfc4000fb2cc02f60241f3e0e99582134aef/urlparse4-0.1.tar.gz#sha256=118b336f5c56e8419c20507710aaaf264aad17d46dd54e580e447f653e401f91 (from https://pypi.org/simple/urlparse4/), version: 0.1
    Found link https://files.pythonhosted.org/packages/90/c1/caaff08824a5237d4d85bf569565049729aa497f1c6c2e4da7e873b82783/urlparse4-0.1.1.tar.gz#sha256=cc69218746b5cfda3543080e5b9dc1e93307222ab343ce729493c3ed292da207 (from https://pypi.org/simple/urlparse4/), version: 0.1.1
    Found link https://files.pythonhosted.org/packages/d4/a9/1a1408217d91d726d7f91d62dae3e33f5728fa07cef51b6d584002c96c80/urlparse4-0.1.2.tar.gz#sha256=048cc4519a49c0bb35ee81636d1f0e0fe2207e3bc1723a6398557fb613092bbd (from https://pypi.org/simple/urlparse4/), version: 0.1.2
    Found link https://files.pythonhosted.org/packages/af/6f/a2d1a397b47ce3af6c5bb8936a7a8f930bf29b4df42081da842c5c84c1d1/urlparse4-0.1.3.tar.gz#sha256=ac0e9869b96a162ceeb175200d4c2f947c42991cfc2d53b8030748facbf8f791 (from https://pypi.org/simple/urlparse4/), version: 0.1.3
  Using version 0.1.3 (newest of versions: 0.1.3, 0.1.2, 0.1.1, 0.1)
  Downloading from URL https://files.pythonhosted.org/packages/af/6f/a2d1a397b47ce3af6c5bb8936a7a8f930bf29b4df42081da842c5c84c1d1/urlparse4-0.1.3.tar.gz#sha256=ac0e9869b96a162ceeb175200d4c2f947c42991cfc2d53b8030748facbf8f791 (from https://pypi.org/simple/urlparse4/)
  Running setup.py (path:/tmp/pip_build_root/urlparse4/setup.py) egg_info for package urlparse4
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/urlparse4/setup.py", line 46
        print "urlparse4/gurl.cpp not found and Cython failed to run to recreate it. Please install/upgrade Cython and try again."
                                                                                                                                 ^
    SyntaxError: Missing parentheses in call to 'print'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/urlparse4/setup.py", line 46

    print "urlparse4/gurl.cpp not found and Cython failed to run to recreate it. Please install/upgrade Cython and try again."

                                                                                                                             ^

SyntaxError: Missing parentheses in call to 'print'

----------------------------------------
Cleaning up...
  Removing temporary dir /tmp/pip_build_root...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/urlparse4
Exception information:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 278, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 1230, in prepare_files
    req_to_install.run_egg_info()
  File "/usr/lib/python3/dist-packages/pip/req.py", line 326, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/usr/lib/python3/dist-packages/pip/util.py", line 715, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/urlparse4
